documentation/src/docs/pravega-concepts.md

```diff
@@ -9,364 +9,259 @@ You may obtain a copy of the License at
-grow and shrink to match the behavior of the data input.  The size of any Stream
-is limited only by the total storage capacity made available to the Pravega
-cluster; if you need bigger streams, simply add more storage to your cluster.
+The number of Stream Segments change over time by using the Pravega's Auto Scaling feature as we discussed in the [Auto Scaling](#elastic-streams-auto-scaling) section. The size of any Stream is determined by the storage capacity of the Pravega cluster. More Streams can be obtained by increasing the storage of the Pravega cluster.
 
-Applications can react to changes
-in the number of Segments in a Stream, adjusting the number of Readers within a
-ReaderGroup, to maintain optimal read parallelism if resources allow.  This is
-useful, for example in a Flink application, to allow Flink to increase or
-decrease the number of task instances that are processing a Stream in parallel,
-as scale events occur over time.
+Applications can react to changes in the number of Segments in a Stream by adjusting the number of Readers within a Reader Group to maintain optimal read parallelism. As a cool use case, Pravega may allow Flink to increase or decrease the number of task instances that are processing a Stream in parallel.
 
 ### Ordering Guarantees
 
-A stream comprises a set of segments that can change over time. Segments that overlap in their area of keyspace have a defined order.
+A Stream comprises a set of Segments that can change over time. Segments that overlap in their area of key space have a defined order.
 
-An event written to a stream is written to a single segment and it is totally ordered with respect to the events of that segment. The existance and position of an event within a segment is strongly consistent.
+An Event written to a Stream is written to a single Segment, and is ordered with respect to the Events of that Segment. The existence and position of an Event within a Segment is strongly consistent.
 
-Readers can be assigned multiple parallel segments (from different parts of keyspace). A reader reading from multiple segments will interleave the events of the segments, but the order of events per segment respects the one of the segment. Specifically, if s is a segment, events e~1 and e~2 of s are such that e~1 precedes e~2, and a reader reads both e~1 and e~2, then the reader will read e~1 before e~2.
+Readers can be assigned multiple parallel Segments (from different parts of key space). A Reader reading from multiple Segments will interleave the Events of the Segments, but the order of Events per Segment is retained. Specifically, if **s** is a Segment, and **s** contains two Events _i.e.,_ **s** **=** {**e~1**,**e~2**} where **e~1** precedes **e~2**. Thus when a Reader tries to read both the Events (**e~1** and **e~2**) the read order is guaranteed by assuring that, the Reader is allowed to read **e~1** before **e~2**.
```

Comment:
Please, rephrase `Thus when a Reader...` -> `Thus, for a Reader reading Segment s, it is guaranteed that **e~1** will be read before **e~2**.`

Reply:
accepted